lib/generators/utils/tooltip.js

```diff
@@ -17,13 +17,18 @@ module.exports = {
  *
  */`;
 	},
-	commonsChunk: _ => {
+	splitChunks: _ => {
 		return `/*
- * We've enabled commonsChunkPlugin for you. This allows your app to
- * load faster and it splits the modules you provided as entries across
- * different bundles!
+ * SplitChunksPlugin is enabled by default and replaced
+ * deprecated CommonsChunkPlugin. It automatically identifies modules which
+ * should be splitted of chunk by heuristics using module duplication count and
+ * module category (i. e. node_modules). And splits the chunks…
  *
- * https://webpack.js.org/plugins/commons-chunk-plugin/
+ * It is safe to remove "splitChunks" from the generated configuration
+ * and was added as an educational example.
+ *
+ * Documentation page will be available here soon:
```

Comment:
We will need to rephrase this a little once [#1863](https://github.com/webpack/webpack.js.org/pull/1863) gets merged.

Reply:
The documentation is available now. Can you rephrase this?

Reply:
webpack/webpack.js.org#1955 was merged, I'll rephrase this when resolving conflict